UT4MasterServer.Web/src/services/http.service.ts

```diff
@@ -19,13 +19,14 @@ export default class HttpService {
         return form;
     }
 
-    async send<K = unknown, T = unknown>(url: string, options?: HttpRequestOptions<T>, method: HttpMethod = 'GET'): Promise<K> {
+    async send<K = unknown, T = unknown>(url: string, options?: HttpRequestOptions<T>, method: HttpMethod = 'GET', form = true): Promise<K> {
         const fetchOptions: RequestInit = { method };
         if (options?.body) {
-            fetchOptions.body = this.formEncode(options.body);
+            fetchOptions.body = form ? this.formEncode(options.body) : JSON.stringify(options.body);
         }
 
-        const headers: HeadersInit = { 'Content-Type': 'application/x-www-form-urlencoded' };
+        const headers: HeadersInit = { 'Content-Type': form ? 'application/x-www-form-urlencoded' : 'application/json' };
+
```

Comment:
Double line
```suggestion
        const headers: HeadersInit = { 'Content-Type': form ? 'application/x-www-form-urlencoded' : 'application/json' };
```

Reply:
I just autoformat with vscode, it wraps when it deems the line too long.

Reply:
I'll add prettier in a separate PR, that should alleviate all of your whitespace concerns.

Reply:
> I just autoformat with vscode, it wraps when it deems the line too long.

I selected 2 lines and deleted blank one in suggestion, but GH don't get it...
You have double lines - 29 and 30